tools/cluster-deploy/cluster-deploy.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 var (
-	ocsRegistryImage       = flag.String("ocs-registry-image", "", "The ocs-registry container image to use in the deployment")
+	ocsCatalogImage        = flag.String("ocs-catalog-image", "", "The ocs catalog image to use in the deployment")
```

Comment:
This text is different from the other instances where you have "ocs catalog container image". Depending on your thoughts on my earlier comment, probably have this one match all the others?